src/components/input/CInputProcessing.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <b-button
+        :variant="variant"
+        v-on="$listeners"
+        @click="onExport"
```

Comment:
This is a general button and not just for exporting, no need to have any @click since we have on listeners.